WebDriverAgentLib/Commands/FBFindElementCommands.m

```diff
@@ -103,6 +103,10 @@ + (NSArray *)routes
 
 + (XCUIElement *)elementUsing:(NSString *)usingText withValue:(NSString *)value under:(XCUIElement *)element
 {
+  if ([usingText isEqualToString:@"xpath"]) {
```

Comment:
I've intentionally created separate method to get only the first xpath match, because the current approach resolves multiple snapshots even if this not needed.

Current implementation in master requires the equal amount of time to return a result for

```
driver.findElement(locator)
```

and

```
driver.findElements(locator)
```

In my implementation the first method call could be much faster, because it only resolves the very first matched snapshot and skips everything else, which saves a lot of time (since _resolve_ is the slowest call in the chain).